includes/razorpay-subscriptions.php

```diff
@@ -353,8 +439,13 @@ protected function getSubscriptionSessionKey($orderId)
         return self::RAZORPAY_SUBSCRIPTION_ID . $orderId;
     }
 
-    protected function getRazorpayApiInstance()
+    protected function setRazorpayApiInstance($keyId, $keySecret)
```

Comment:
don't see the point of refactoring this into 2 separate methods. Atleast move them closer to the constructor, if you think they're necessary.

Reply:
I just think that they are good helper methods to have.

Reply:
Removed both methods